numba/core/ir.py

```diff
@@ -1175,8 +1175,11 @@ def walk(thename):
         return vers
 
     def make_temp(self, loc):
+        return self.make_temp_with_prefix('', loc)
+
+    def make_temp_with_prefix(self, prefix, loc):
```

Comment:
Is a new method needed? Would adding a `prefix` kwarg to the existing `make_temp` have the same/similar effect?